src/mappings/_utils.test.ts

```diff
@@ -194,16 +283,26 @@ describe("_utils", () => {
             testHelpers.getForeignAssetsCache().set(testAssetId, fakeAsset);
             
             const testForeignAsset = new ForeignAsset({asset: testAssetId});
-            const actualSymbol = await symbolFromCurrency(testForeignAsset);
+            const actualSymbol = await tickerFromCurrency(testForeignAsset);
             expect(actualSymbol).toBe(fakeAsset.ticker);
         });
 
+        it("should return ticker for lend token", async () => {
+            const testLendTokenId = fakeLendToken.lendToken.id;
+            // prepare cache for lookup
+            testHelpers.getLendTokensCache().set(testLendTokenId, fakeLendToken);
+            
+            const testForeignAsset = new LendToken({lendTokenId: testLendTokenId});
+            const actualSymbol = await tickerFromCurrency(testForeignAsset);
```

Comment:
Probably forgot to rename?
```suggestion
            const testLendToken = new LendToken({lendTokenId: testLendTokenId});
            const actualSymbol = await tickerFromCurrency(testLendToken);
```

Reply:
Good catch, thanks!